llvm/include/llvm/ObjectYAML/DXContainerYAML.h

```diff
@@ -56,7 +56,7 @@ struct DXILProgram {
   std::optional<std::vector<llvm::yaml::Hex8>> DXIL;
 };
 
-#define SHADER_FLAG(Num, Val, Str) bool Val = false;
+#define SHADER_FEATURE_FLAG(Num, Val, Str) bool Val = false;
 struct ShaderFlags {
```

Comment:
Should this also be `FeatureFlags`?

Reply:
There're 39 DxilModelFlags and 33 FeatureFlags in dxc currently.
Maybe we should save DxilModelFlags and generate FeatureFlags from DxilModelFlags?

Reply:
Updated to ShaderFeatureFlags.